app/javascript/components/rooms/FeatureTabs.jsx

```diff
@@ -9,16 +9,16 @@ export default function FeatureTabs() {
   return (
     <Row className="pt-5 mx-0">
       <Tabs defaultActiveKey="recordings">
-        <Tab eventKey="recordings" title="Recordings">
+        <Tab eventKey="recordings" title="Recordings" unmountOnExit>
           <RoomRecordings />
         </Tab>
-        <Tab eventKey="presentation" title="Presentation">
+        <Tab eventKey="presentation" title="Presentation" unmountOnExit>
           <Presentation />
         </Tab>
-        <Tab eventKey="access" title="Access">
+        <Tab eventKey="access" title="Access" unmountOnExit>
           <SharedAccess />
         </Tab>
-        <Tab eventKey="settings" title="Settings">
+        <Tab eventKey="settings" title="Settings" unmountOnExit>
```

Comment:
This prevents the queries from firing until the tab comes into view.

What ti was doing before was firing all queries for all tabs as soon as the first tab loads

Reply:
Couldn't we achieve the same thing by setting the `unmountOnExit` prop on the `Tabs` level?

Reply:
Yep you're right